src/main/groovy/com/bmuschko/gradle/docker/utils/DockerThreadContextClassLoader.groovy

```diff
@@ -422,6 +422,16 @@ class DockerThreadContextClassLoader implements ThreadContextClassLoader {
         enhancer.create()
     }
 
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    def createExecCallback(OutputStream out, OutputStream err) {
```

Comment:
Lets follow the model set by method `createLoggingCallback` instead of piping to stdout.